### PR TITLE
[4.0] ChipList docs

### PR DESCRIPTION
**Pull request by svdimitr**

### CODE REVIEW

File: components/chiplist/selection.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        public string Text { get; set; }
+        public FontIcon? Icon { get; set; }
+    }
+}
+````
+
+
+## See Also
+
+* [ChipList Events]({%slug ChipList-events%})
```
Comment (ntacheva) [inline components/chiplist/selection.md:106]:
```suggestion
* [ChipList Events]({%slug chiplist-events%})
```

File: components/chiplist/overview.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+position: 0
+---
+
+# Blazor ChipList Overview
+
+The <a href="https://www.telerik.com/blazor-ui/chiplist" target="_blank">Blazor ChipList component</a> shows pieces of information in a compact form. The individual chips can be [selected]({%slug chiplist-selection%}), removed, or disabled. You can respond to various user interactions through the exposed [events]({%slug chiplist-events%}), customize the [appearance]({%slug chiplist-appearance%}) of the chips, or define custom content for the chip with the [ItemTemplate]({%slug chiplist-templates%}).
+
+## Creating Blazor ChipList
+
+To use a Telerik ChipList for Blazor:
```
Comment (ntacheva) [inline components/chiplist/overview.md:17]:
```suggestion
```

File: components/chiplist/overview.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+The table below lists the ChipList parameters. Also check the [ChipList API Reference](/blazor-ui/api/Telerik.Blazor.Components.TelerikChipList) for all parameters, methods and events.
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+| Parameter | Type and Default&nbsp;Value | Description |
+| --- | --- | --- |
+| `AriaLabel` | `string` | Maps to the `aria-label` attribute. Use  this parameter if the text that labels the component is not visible. |
+| `AriaLabelledBy` | `string` | Maps to the `area-labelledby` attribute. Use this parameter to reference another element to define its accessible name. |
+| `Class` | `string` | An additional CSS class for the `<div class="k-chip-list">` element. Use it to [customize the component styles and override the theme]({%slug themes-override%}). |
```
Comment (ntacheva) [inline components/chiplist/overview.md:84]: I suggest moving the `Class` parameter and the appearance section to a dedicated "Styling and appearance" subsection. Ref: https://docs.telerik.com/blazor-ui/components/autocomplete/overview#styling-and-appearance

Comment (svdimitr) [reply components/chiplist/overview.md:83]: I am a bit against that. The class can be used for more things than simply styling the component. The Aria* is accessibility related, did I miss something in the comment?

**Review by ntacheva — CHANGES_REQUESTED** [review]
Add the ChipList to the _config.yml

**Comment by ntacheva** [comment]
I suggest moving the Events article after the Templates for consistency.

**Review by ntacheva — APPROVED** [review]

**Pull request merged**